drivers/src/soc_ifc.rs

```diff
@@ -131,6 +132,85 @@ impl SocIfc {
         let flags: MfgFlags = soc_ifc_regs.cptra_dbg_manuf_service_reg().read().into();
         flags.contains(MfgFlags::GENERATE_IDEVID_CSR)
     }
+
+    /// Enbale or disable WDT1
+    ///
+    /// # Arguments
+    /// * `enable` - Enable or disable WDT1
+    ///
+    pub fn configure_wdt1(&mut self, enable: bool) {
+        let soc_ifc_regs = self.soc_ifc.regs_mut();
+        soc_ifc_regs
+            .cptra_wdt_timer1_en()
+            .write(|w| w.timer1_en(enable));
+    }
+
+    fn get_cycle_count(&self, seconds: u32) -> CaliptraResult<u64> {
+        const PICO_UNIT: u64 = 0xE8D4A51000;
+        let clock_period_picosecs = self.soc_ifc.regs().cptra_timer_config().read() as u64;
+        if clock_period_picosecs == 0 {
+            Err(CaliptraError::DRIVER_SOC_IFC_INVALID_TIMER_CONFIG)
+        } else {
+            Ok((seconds as u64) * (PICO_UNIT / clock_period_picosecs))
```

Comment:
This division is bloating the ROM size due to compiler_builtins::int::specialized_div_rem::u64_div_rem

Reply:
Switched to a 32-bit division.